CMakeLists.txt

```diff
@@ -22,12 +22,29 @@ add_definitions(-DPROJECT_CODENAME="${PROJECT_CODENAME}")
 add_definitions(-DPROJECT_QUALITY="${PROJECT_QUALITY}")
 add_definitions(-DPROJECT_SERIES="${PROJECT_SERIES}")
 
-set (LIB_SUFFIX "" CACHE STRING "Define suffix of directory name (32/64)" )
-set (LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}" )
-set (PLUGIN_INSTALL_PATH "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}")
-set (INCLUDE_INSTALL_DIR "${CMAKE_INSTALL_INCLUDEDIR}")
-set (DOC_INSTALL_DIR "${CMAKE_INSTALL_DOCDIR}/${PROJECT_SERIES}")
-set (PLUGIN_SEGMENT_INSTALL_PATH "${CMAKE_INSTALL_SYSCONFDIR}/ambd/plugins.d")
+if(NOT DEFINED LIB_SUFFIX)
```

Comment:
You are right. I also agree with your opinion. IIRC, only few open sources use this way (i.e ${prefix}/@INCLUDE_INSTALL_DIR@), but it works without any errors and I like it.